APP-837 / APP-838

robson-sanchez opened the pull request:
- APP-837 Remove Symphony dependencies from sdk
- APP-838 Remove Symphony dependencies from webcommons

robson-sanchez committed:
APP-838 Remove Symphony dependencies from webcommons

robson-sanchez committed:
APP-837 Remove Symphony dependencies from sdk

ecarrenho-daitan commented on integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/client/model/Envelope.java:
```diff
@@ -0,0 +1,32 @@
+package org.symphonyoss.integration.provisioning.client.model;
+
+/**
+ * Wrap the data into an envelope.
```
Add information here to explain that the Apps Repository API wraps all request and response content in a "data" object and that is the purpose of this wrapper class.

robson-sanchez replied:
FIXED

ecarrenho-daitan reviewed (commented):
LGTM.

ecarrenho-daitan commented:
@dnsymphony please review.

dnsymphony commented on integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/client/AppRepositoryClient.java:
```diff
@@ -62,133 +48,91 @@
-    config.setKeymanagerUrl(properties.getKeyManagerUrl());
-    config.setSymphonyUrl(properties.getSymphonyUrl());
-    config.setLoginUrl(properties.getLoginUrl());
+  public List getAppsAvailable(String userId) throws AppRepositoryClientException {
```
Please add javadoc to all methods

robson-sanchez replied:
FIXED

dnsymphony reviewed (changes requested):
Unit tests?

robson-sanchez committed:
APP-837 Remove Symphony dependencies from sdk

- Fix PR comments

robson-sanchez committed:
APP-837 Remove Symphony dependencies from sdk

- Unit tests

robson-sanchez commented:
@dnsymphony I addressed the PR's comments and also added some unit tests to the module integration-provisioning.

The missing unit tests will be implemented in my next PR related to the APP-836

Please check again if everything is ok.

robson-sanchez commented:
I'm gonna open another PR directly on the symphonyoss
